payment.html

```diff
@@ -49,7 +49,7 @@
     <main>
       <div class="py-5 text-center">
         
-        <h2>Checkout form</h2>
+        <h2>Checkout</h2>
```

Comment:
Where is the navbar , I can't see any changes you have made regarding the navbar?